setup.py

```diff
@@ -33,7 +33,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://jovian.ml/?utm_source=pypi",
-    packages=setuptools.find_packages(),
+    packages=setuptools.find_packages(exclude=["*.tests", "*.tests.*"]),
```

Comment:
can we just do : `exclude=["tests"]` ?

Reply:
I tried that, it did not work. There were many cases in stackoverflow too, where it did not work.
Found this on the docs, to exclude any sub package named tests https://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages